client/my-sites/plugins/main.jsx

```diff
@@ -442,7 +442,7 @@ export class PluginsMain extends Component {
 		const content = (
 			<>
 				<DocumentHead title={ pageTitle } />
-				<QueryJetpackPlugins siteIds={ this.props.siteIds } />
+				<QueryAllJetpackPlugins />
```

Comment:
It feels like overkill to me to request the plugins for all sites (and update plugins on all autoupdate-enabled sites) when accessing the plugins page of a single site (`/plugins/manage/:site`). WDYT?

Reply:
Thanks! Fixed in fba75ea7f0cf023bc061edcf7975cc5c65142abb